slosilo.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |gem|
   gem.name          = "slosilo"
   gem.require_paths = ["lib"]
   gem.version       = Slosilo::VERSION
-  gem.required_ruby_version = '>= 1.9.3'
+  gem.required_ruby_version = '>= 3.0.0'
```

Comment:
According this line changelog and readme should be re-phrased in terms of switch to ruby 3 rather support

Reply:
Thanks - Updated README & CHANGELOG files.